Capri.Web/Controllers/ProposalController.cs

```diff
@@ -117,6 +117,11 @@ await _submittedProposalGetter
         [HttpPost]
         public async Task<IActionResult> Create([FromBody] ProposalRegistration registration)
         {
+            if(registration == null || !ModelState.IsValid)
+            {
+                return BadRequest("You provided invalid data");
```

Comment:
Same as here, impersonal form will be better.

Reply:
Done